CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0 (2019-01-10)
```

Comment:
not that it really matters, but its the Eleventh! :)

Reply:
Ooh, good catch. My notepad still said 10 from yesterday ;p